tests/e2e/test_drop_in_environments.py

```diff
@@ -186,20 +202,25 @@ def r_regression_custom_model(self, r_drop_in_env):
         )
 
     @pytest.mark.parametrize(
-        "model, test_data_id",
+        "model, test_data_id, max_wait",
         [
-            ("python311_genai_custom_model", "regression_testing_data"),
-            ("r_regression_custom_model", "regression_testing_data"),
-            ("torch_regression_custom_model", "regression_testing_data"),
-            ("keras_regression_custom_model", "regression_testing_data"),
-            ("xgb_regression_custom_model", "regression_testing_data"),
-            ("onnx_regression_custom_model", "regression_testing_data"),
-            ("sklearn_regression_custom_model", "regression_testing_data"),
-            ("sklearn_fips_regression_custom_model", "regression_testing_data"),
-            ("java_regression_custom_model", "regression_testing_data"),
+            ("python311_genai_custom_model", "regression_testing_data", 2 * DEFAULT_MAX_WAIT),
```

Comment:
Now we just need keep an eye on the node where this test runs.

Usually any problems we had with these tests were related to the node resources.